### ISSUE TITLE
How to manually start the video

### ISSUE THREAD

**Issue by ghost** [mbebenita/Broadway#69]
Is it possible to add a click event to start the video. There's broadway.play() but it doesn't seem to work. Is there any good documentation explaining the decoder and how to use it. I'm kinda new to this would really appreciate some help.

**Comment by soliton4** [comment]
hi @chaitanyapro 
there is a api documentation in the readme

in order to improve the documentation it might be helpful to know at what point the docs didnt help you any more

**Comment by soliton4** [comment]
@chaitanyapro we are actually actively trying to make this a better thing and on larger scope to make the world a better place.
but we need your help!

**Comment by chaitanyapro** [comment]
I think you should also focus on newbies. The documentation is quite confusing. That's the only reason why I migrated my project to ogv.js
An amazing script with not so amazing documentation

**Comment by soliton4** [comment]
thx for the feedback!
can you rephrase it in a constructive way please

**Comment by erkkimon** [comment]
This is what I've searched for 6 months, and now I suddenly bumped into this. JS-based decoding solves so many problems. Thank you so much!

Also I have difficulties to understand everything written in the source code of demos, which are very nice and simple. I have been able to succeed through trying and failing. It would be wonderful to see a example for dummies about how to add playback controls to Broadway video.

I was also wondering if there's any way to pause the video. I did some grepping in the source code, and found out that there is nothing that I would associate with pause functionality. Would it be difficult to implement?

Thanks for this great contribution for web-based app development!

**Comment by mbebenita** [comment]
What parts were difficult to understand? I believe that if you use the Player API directly and don't call `decode` then the video should stop.

**Comment by soliton4** [comment]
@mbebenita i think ppl just want a out of the box solution that works even better than what browsers offer.
i kinda understand that but i dont have the time to pull that off at the mom. especially since i dont have any personal need for it.
